login.css

```diff
@@ -88,12 +88,12 @@ ul li ul{
 }
 /* Set up the footer */
 .footer{
-	background-color: #C8A2C8;
+	background-color: #0C59CF;
 	grid-column-start: 1;
 	grid-column-end: span 4;
 	text-align: center;
 	height:35px;
-	color:black;
+	color:white;
```

Comment:
06/08/2021: I make the color font for the text in the footer to be white because it white and navy blue was a good combination that will work well together better than black.